vignettes/layer.Rmd

```diff
@@ -16,4 +16,301 @@ knitr::opts_chunk$set(
 
 ```{r setup}
 library(tlang)
+library(dplyr)
+```
+
+A core design element of `tlang` is to offer the ability to layer and build template table formats that can be shared, improved, and eventually customized for a specific table. The idea being that organizations can develop a standard `tfrmt` object that can be reused across many different tables, and make it so programmers can focus on setting up the data as opposed to formatting of the table. 
+
+There are two ways that `tfrmt` objects may be layered. The preferred method is through the first argument of `tfrmt` - `tfrmt_obj`. The second, but perhaps more explicit method is using the `layer_tfrmt` function. We will go through their applications below.  
```

Comment:
It might be good to describe these in more laymans terms and then use the same  description in the rest of the doc.

Reply:
I agree I think the information about layer_tfrmt needs to be moved out of here and into the help docs for layer_tfrmt

Reply:
I agree I think the information about layer_tfrmt needs to be moved out of here and into the help docs for layer_tfrmt